autometa/common/coverage.py

```diff
@@ -60,16 +60,33 @@ def from_spades_names(records):
         Why the exception is raised.
```

Comment:
Item 1 in issue #21 not addressed.

Reply:
The function is 7 lines and (I think) reads almost like a sentence. 

I.e. "for record id in records split cov and take the last element."

 An example header is also located in the description for the split to be interpreted with the example above. The name of the series is specified and the index name is assigned until returned. I'll remove the exception docstring. Otherwise, how else would I make this simpler without being periphrastic?

Reply:
The description `Autometa coverage` at the beginning of the script does not seem very descriptive to me.